src/api/app/controllers/status/reports_controller.rb

```diff
@@ -49,7 +49,9 @@ def require_checkable
   end
 
   def xml_hash
```

Comment:
where is this method used? :thinking:

Reply:
Not used anymore, thanks, good catch :+1: